src/integration-test/java/com/gradle/exportapi/BuildDaoIntegrationTest.java

```diff
@@ -0,0 +1,35 @@
+package com.gradle.exportapi;
+
+import com.gradle.exportapi.dao.BuildDAO;
+import com.gradle.exportapi.model.Build;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.Instant;
+
+public class BuildDaoIntegrationTest extends DatabaseBaseIntegrationTest {
+
+    @Before
+    public void createDb() {
+        // Assuming the database has already been created
+        // recreateDb();
```

Comment:
Usually you don't need to run this, if the DB already exists.  But if it hasn't been run yet then it's handy.  This will be removed when I set up CI in #15